### DISCUSSION TITLE
Id icsharpcode.sharpziplib.dll com visible?

### DISCUSSION THREAD

**Discussion by WaterskiScoring** [icsharpcode/SharpZipLib#846] [Q&A] [answered]
I need a tool to read zip files in a class ASP page.  Can this library be used for this purpose?

**Comment by piksel** [comment, marked as answer]
I think so. Not exactly sure how you mean to use it, but it sounds like it would work.
If you actually mean ASP (not .NET), you might have to use an older version though...

**Reply by WaterskiScoring** [reply]
Thank you for the response.  Yes I meant classic ASP running on IIS 10.0.  One additional questions is the DLL COM visible because that is the key to being able to use it with ASP
